bindata/v3.11.0/kube-scheduler/pod.yaml

```diff
@@ -58,7 +58,17 @@ spec:
   hostNetwork: true
   priorityClassName: system-node-critical
   tolerations:
-  - operator: "Exists"
+   - key: node-role.kubernetes.io/master  # Just tolerate NoSchedule taint on master node. If there are other conditions like disk-pressure etc, let's not schedule the control-plane pods onto that node.
+     operator: Exists
+     effect: "NoSchedule"
+   - key: "node.kubernetes.io/unreachable"
+     operator: "Exists"
+     effect: "NoExecute"
+     tolerationSeconds: 120 # Evict pods within 2 mins.
```

Comment:
This should help in evicting within 2 min @smarterclayton

Reply:
This is good.  How do we get this to all other operators (once we're clear here that this works the way we expect)?

Reply:
I tested this manually on my cluster. @sjenning and I plan to create PRs against all existing components.